pages/toast.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../styles/main.css" />
+    <title>SnackBar Component</title>
+</head>
+
+<body>
+    <div class="wrapper">
+        <div class="bar-wrapper">
+            <div class="content">Can't send photo. Retry in 5 seconds</div>
+            <button class="action-btn">RETRY</button>
+            <button class="close-btn">X</button>
```

Comment:
content name seems very generic, add something like toast-content.
Also use X icon instead of the letter itself.